Dockerfile

```diff
@@ -1,5 +1,11 @@
 FROM snapcore/snapcraft@sha256:6d771575c134569e28a590f173f7efae8bf7f4d1746ad8a474c98e02f4a3f627
 
+# Faster mirrors
+RUN echo "deb mirror://mirrors.ubuntu.com/mirrors.txt trusty main restricted universe multiverse" > /etc/apt/sources.list && \
+    echo "deb mirror://mirrors.ubuntu.com/mirrors.txt trusty-updates main restricted universe multiverse" >> /etc/apt/sources.list && \
+    echo "deb mirror://mirrors.ubuntu.com/mirrors.txt trusty-security main restricted universe multiverse" >> /etc/apt/sources.list && \
```

Comment:
Rather than blowing away the sources.list here, would it be better to `sed -i 's/http://archive.ubuntu.com/mirror://mirrors.ubuntu.com/mirrors.txt/g'`?  That way if source.list contains more lines other than these 3 in the future, it'll still work?  Not sure... might be a toss up.